precice/_version.py

```diff
@@ -0,0 +1,525 @@
+
+# This file helps to compute a version number in source trees obtained from
+# git-archive tarball (such as those provided by githubs download-from-tag
+# feature). Distribution tarballs (built by setup.py sdist) and build
+# directories (produced by setup.py build) will contain a much shorter file
+# that just contains the computed version number.
+
+# This file is released into the public domain. Generated by
+# versioneer-0.19 (https://github.com/python-versioneer/python-versioneer)
+
+"""Git implementation of _version.py."""
```

Comment:
Does this file give us Git as a dependency for the package? Will it work if you use the tarball instead of the `python-bindings` Git repository?

Reply:
Good point. I will try this out later.

Reply:
I checked the `precice.__version__` on a system without git and it worked. This was a bit surprising, since in the source-code git is actually called. But the versioneer is doing something clever here. Looking at `$HOME/.local/lib/python3.8/site-packages/precice/_version.py` I found out that the version is hard-coded and no git is needed anymore by the user. Looks good :smile: